LPD-54584 Update path to common.sh

igor-franca opened the pull request:
References:

- Parent Task: [LPD-54584](https://liferay.atlassian.net/browse/LPD-54584) Playwright - Fix path to common.sh in playwright tests
- Recent pull request related: https://github.com/brianchandotcom/liferay-portal/pull/161394

[LPD-54584]: https://liferay.atlassian.net/browse/LPD-54584?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

liferay-continuous-integration commented:
<html><h3>:x: ci:test:playwright - 96 out of 134 jobs passed in 1 hour 54 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/1335">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1407401_14341</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/319f4541315aa345aed334837345be457df28c75">319f4541315aa345aed334837345be457df28c75</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/83f09f7f71c3580c1ebb4fe81083373c471d43b1">83f09f7f71c3580c1ebb4fe81083373c471d43b1</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/207322413">EE Development Acceptance (master) - 1334 - 2025-05-02[00:35:13]</a></p><details><summary><strong>ci:test:playwright - 96 out of 134 jobs PASSED</strong></summary><div><h4>38 Failed Jobs:</h4><ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/14341/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>playwright-compile<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338188/">playwright-compile/0/0</a></li></ul></li><li>playwright-js-tomcat90-postgresql163<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238628/">playwright-js-tomcat90-postgresql163/0/0</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1165102/"><strike>playwright-js-tomcat90-postgresql163/1/0</strike></a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1165104/">playwright-js-tomcat90-postgresql163/2/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1152087/">playwright-js-tomcat90-postgresql163/3/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1119222/">playwright-js-tomcat90-postgresql163/4/0</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1058768/">playwright-js-tomcat90-postgresql163/5/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1111287/"><strike>playwright-js-tomcat90-postgresql163/6/0</strike></a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1152086/">playwright-js-tomcat90-postgresql163/7/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/813760/">playwright-js-tomcat90-postgresql163/8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1119224/">playwright-js-tomcat90-postgresql163/9/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1061100/">playwright-js-tomcat90-postgresql163/10/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1152095/">playwright-js-tomcat90-postgresql163/11/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1061096/">playwright-js-tomcat90-postgresql163/12/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1056745/">playwright-js-tomcat90-postgresql163/13/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1132659/">playwright-js-tomcat90-postgresql163/14/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338191/">playwright-js-tomcat90-postgresql163/15/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/913946/"><strike>playwright-js-tomcat90-postgresql163/15/1</strike></a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1395530/">playwright-js-tomcat90-postgresql163/16/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1061094/"><strike>playwright-js-tomcat90-postgresql163/17/0</strike></a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1119221/"><strike>playwright-js-tomcat90-postgresql163/17/1</strike></a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238630/">playwright-js-tomcat90-postgresql163/18/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1257094/">playwright-js-tomcat90-postgresql163/19/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1061095/">playwright-js-tomcat90-postgresql163/20/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/813762/">playwright-js-tomcat90-postgresql163/21/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1061099/"><strike>playwright-js-tomcat90-postgresql163/22/0</strike></a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238636/"><strike>playwright-js-tomcat90-postgresql163/23/0</strike></a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/813768/"><strike>playwright-js-tomcat90-postgresql163/24/0</strike></a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1231149/">playwright-js-tomcat90-postgresql163/25/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238625/"><strike>playwright-js-tomcat90-postgresql163/26/0</strike></a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1152092/">playwright-js-tomcat90-postgresql163/27/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238629/"><strike>playwright-js-tomcat90-postgresql163/28/0</strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338190/"><strike>playwright-js-tomcat90-postgresql163/29/0</strike></a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1132656/">playwright-js-tomcat90-postgresql163/30/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1111284/">playwright-js-tomcat90-postgresql163/31/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1395514/">playwright-js-tomcat90-postgresql163/32/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1119227/">playwright-js-tomcat90-postgresql163/33/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338186/"><strike>playwright-js-tomcat90-postgresql163/34/0</strike></a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/813761/">playwright-js-tomcat90-postgresql163/35/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/913941/"><strike>playwright-js-tomcat90-postgresql163/36/0</strike></a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/913944/">playwright-js-tomcat90-postgresql163/37/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238632/"><strike>playwright-js-tomcat90-postgresql163/38/0</strike></a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1056750/">playwright-js-tomcat90-postgresql163/39/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1111285/">playwright-js-tomcat90-postgresql163/40/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1119226/"><strike>playwright-js-tomcat90-postgresql163/41/0</strike></a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1152088/">playwright-js-tomcat90-postgresql163/42/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1395510/"><strike>playwright-js-tomcat90-postgresql163/43/0</strike></a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1061097/"><strike>playwright-js-tomcat90-postgresql163/44/0</strike></a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/813765/">playwright-js-tomcat90-postgresql163/45/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1152085/">playwright-js-tomcat90-postgresql163/46/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1231147/">playwright-js-tomcat90-postgresql163/47/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/913945/"><strike>playwright-js-tomcat90-postgresql163/48/0</strike></a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238631/">playwright-js-tomcat90-postgresql163/48/1</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1058766/">playwright-js-tomcat90-postgresql163/49/0</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1058767/">playwright-js-tomcat90-postgresql163/50/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1061098/"><strike>playwright-js-tomcat90-postgresql163/51/0</strike></a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1111288/">playwright-js-tomcat90-postgresql163/52/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1202411/">playwright-js-tomcat90-postgresql163/53/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1119223/">playwright-js-tomcat90-postgresql163/54/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/813764/">playwright-js-tomcat90-postgresql163/55/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/813766/"><strike>playwright-js-tomcat90-postgresql163/56/0</strike></a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238627/"><strike>playwright-js-tomcat90-postgresql163/57/0</strike></a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1111290/">playwright-js-tomcat90-postgresql163/58/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338187/">playwright-js-tomcat90-postgresql163/59/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1257095/"><strike>playwright-js-tomcat90-postgresql163/60/0</strike></a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/813769/">playwright-js-tomcat90-postgresql163/61/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1231146/"><strike>playwright-js-tomcat90-postgresql163/62/0</strike></a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1132663/">playwright-js-tomcat90-postgresql163/63/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1231152/"><strike>playwright-js-tomcat90-postgresql163/64/0</strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338197/">playwright-js-tomcat90-postgresql163/65/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1202412/">playwright-js-tomcat90-postgresql163/66/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1395512/">playwright-js-tomcat90-postgresql163/67/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1395515/">playwright-js-tomcat90-postgresql163/68/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/913947/">playwright-js-tomcat90-postgresql163/68/1</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1152094/">playwright-js-tomcat90-postgresql163/69/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1119225/"><strike>playwright-js-tomcat90-postgresql163/70/0</strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338193/">playwright-js-tomcat90-postgresql163/71/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338195/"><strike>playwright-js-tomcat90-postgresql163/72/0</strike></a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1056743/">playwright-js-tomcat90-postgresql163/73/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1202415/">playwright-js-tomcat90-postgresql163/74/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/913950/">playwright-js-tomcat90-postgresql163/75/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/913949/">playwright-js-tomcat90-postgresql163/76/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/913943/"><strike>playwright-js-tomcat90-postgresql163/77/0</strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338196/">playwright-js-tomcat90-postgresql163/78/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1395516/">playwright-js-tomcat90-postgresql163/79/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1152089/"><strike>playwright-js-tomcat90-postgresql163/80/0</strike></a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1111283/">playwright-js-tomcat90-postgresql163/81/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1257093/">playwright-js-tomcat90-postgresql163/82/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1202416/">playwright-js-tomcat90-postgresql163/83/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1061104/">playwright-js-tomcat90-postgresql163/84/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1132657/">playwright-js-tomcat90-postgresql163/85/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1056744/">playwright-js-tomcat90-postgresql163/86/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1056746/"><strike>playwright-js-tomcat90-postgresql163/87/0</strike></a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238639/">playwright-js-tomcat90-postgresql163/88/0</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1165100/">playwright-js-tomcat90-postgresql163/89/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1257091/">playwright-js-tomcat90-postgresql163/90/0</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1165099/">playwright-js-tomcat90-postgresql163/91/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1257092/">playwright-js-tomcat90-postgresql163/92/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1111286/">playwright-js-tomcat90-postgresql163/93/0</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1165105/">playwright-js-tomcat90-postgresql163/94/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1231151/">playwright-js-tomcat90-postgresql163/95/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1395511/">playwright-js-tomcat90-postgresql163/96/0</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1165103/"><strike>playwright-js-tomcat90-postgresql163/97/0</strike></a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1056747/">playwright-js-tomcat90-postgresql163/98/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238626/">playwright-js-tomcat90-postgresql163/99/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338192/">playwright-js-tomcat90-postgresql163/100/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1152090/">playwright-js-tomcat90-postgresql163/101/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1119229/">playwright-js-tomcat90-postgresql163/102/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1395513/">playwright-js-tomcat90-postgresql163/103/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1132655/">playwright-js-tomcat90-postgresql163/104/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/813767/">playwright-js-tomcat90-postgresql163/105/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1132658/">playwright-js-tomcat90-postgresql163/106/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1132660/"><strike>playwright-js-tomcat90-postgresql163/107/0</strike></a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1231145/">playwright-js-tomcat90-postgresql163/108/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1152093/"><strike>playwright-js-tomcat90-postgresql163/109/0</strike></a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1202422/"><strike>playwright-js-tomcat90-postgresql163/109/1</strike></a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1202414/"><strike>playwright-js-tomcat90-postgresql163/110/0</strike></a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1132661/">playwright-js-tomcat90-postgresql163/111/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1119230/">playwright-js-tomcat90-postgresql163/112/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1111289/">playwright-js-tomcat90-postgresql163/113/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338189/">playwright-js-tomcat90-postgresql163/114/0</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1058769/">playwright-js-tomcat90-postgresql163/115/0</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1165108/">playwright-js-tomcat90-postgresql163/116/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1111282/">playwright-js-tomcat90-postgresql163/117/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1231148/">playwright-js-tomcat90-postgresql163/118/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1231150/"><strike>playwright-js-tomcat90-postgresql163/119/0</strike></a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1119228/"><strike>playwright-js-tomcat90-postgresql163/120/0</strike></a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1202413/">playwright-js-tomcat90-postgresql163/121/0</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1165101/"><strike>playwright-js-tomcat90-postgresql163/121/1</strike></a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/813763/">playwright-js-tomcat90-postgresql163/122/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1202417/">playwright-js-tomcat90-postgresql163/123/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1152091/">playwright-js-tomcat90-postgresql163/124/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/913942/">playwright-js-tomcat90-postgresql163/125/0</a></li></ul></li></ul></div><details><summary><strong>96 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14341/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/913946//consoleText">playwright-js-tomcat90-postgresql163/15/1</a> - <a href="https://storage.cloud.google.com/testray-results/2025-05/test-1-40/test-portal-acceptance-pullrequest(master)/14341/playwright-js-tomcat90-postgresql163/15/1/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/913946/testReport/account-admin-web_main_accountUsers.spec/ts">account-admin-web/main/accountUsers.spec.ts</a></summary><div><ol><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/913946//testReport/account-admin-web_main_accountUsers.spec/ts/A_user_with_an_invalid_domain_cannot_be_added_to_an_account">A user with an invalid domain cannot be added to an account</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/813768//consoleText">playwright-js-tomcat90-postgresql163/24/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-05/test-1-40/test-portal-acceptance-pullrequest(master)/14341/playwright-js-tomcat90-postgresql163/24/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/813768/testReport/site-admin-web_main_memberships.spec/ts">site-admin-web/main/memberships.spec.ts</a></summary><div><ol><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/813768//testReport/site-admin-web_main_memberships.spec/ts/Bulk_removal_of_roles_from_user_groups">Bulk removal of roles from user groups</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238629//consoleText">playwright-js-tomcat90-postgresql163/28/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-05/test-1-40/test-portal-acceptance-pullrequest(master)/14341/playwright-js-tomcat90-postgresql163/28/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238629/testReport/commerce_commerce-checkout-web_main_checkout.spec/ts">commerce/commerce-checkout-web/main/checkout.spec.ts</a></summary><div><ol><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238629//testReport/commerce_commerce-checkout-web_main_checkout.spec/ts/Checkout_order_detail_redirect_works_correctly_when_order_DPT_is_enabled">Checkout order detail redirect works correctly when order DPT is enabled</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238629//testReport/commerce_commerce-checkout-web_main_checkout.spec/ts/Checkout_widget_configuration_to_display_full_addresses_and_phone_number">Checkout widget configuration to display full addresses and phone number</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238629//testReport/commerce_commerce-checkout-web_main_checkout.spec/ts/Delivery_group_multishipping_checkout_summary">Delivery group multishipping checkout summary</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238629//testReport/commerce_commerce-checkout-web_main_checkout.spec/ts/Payment_Term_is_reset_correctly">Payment Term is reset correctly</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1238629//testReport/commerce_commerce-checkout-web_main_checkout.spec/ts/Set_address_subtype_during_checkout">Set address subtype during checkout</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338186//consoleText">playwright-js-tomcat90-postgresql163/34/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-05/test-1-40/test-portal-acceptance-pullrequest(master)/14341/playwright-js-tomcat90-postgresql163/34/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338186/testReport/commerce_commerce-currency-web_main_commerceAdminCurrency.spec/ts">commerce/commerce-currency-web/main/commerceAdminCurrency.spec.ts</a></summary><div><ol><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338186//testReport/commerce_commerce-currency-web_main_commerceAdminCurrency.spec/ts/COMMERCE_9936_A_disabled_default_currency_should_not_be_usable">COMMERCE-9936 A disabled default currency should not be usable</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338186/testReport/commerce_commerce-currency-web_main_multiCurrency.spec/ts">commerce/commerce-currency-web/main/multiCurrency.spec.ts</a></summary><div><ol><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338186//testReport/commerce_commerce-currency-web_main_multiCurrency.spec/ts/Buyer_can_change_currency_with_active_order">Buyer can change currency with active order</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338186//testReport/commerce_commerce-currency-web_main_multiCurrency.spec/ts/Buyer_can_change_currency_with_active_order_and_product_price_with_price_list_are_converted_accordingly">Buyer can change currency with active order and product price with price list are converted accordingly</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1338186//testReport/commerce_commerce-currency-web_main_multiCurrency.spec/ts/Buyer_can_change_currency_with_currency_selector_and_prices_are_converted_accordingly">Buyer can change currency with currency selector and prices are converted accordingly</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/813766//consoleText">playwright-js-tomcat90-postgresql163/56/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-05/test-1-40/test-portal-acceptance-pullrequest(master)/14341/playwright-js-tomcat90-postgresql163/56/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/813766/testReport/document-library-web_main_fileEntry.spec/ts">document-library-web/main/fileEntry.spec.ts</a></summary><div><ol><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/813766//testReport/document-library-web_main_fileEntry.spec/ts/Error_uploading_multiples_files_with_custom_document_type">Error uploading multiples files with custom document type</a></li></ol></div></details></li></ol></div></li><li>...</li></ol></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/14341/">test-portal-acceptance-pullrequest(master)#14341</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14341/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>playwright<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-bpm/liferay-portal/pull/4460">liferay-bpm#4460</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/207398299">[master] ci:test:playwright - igor-franca &gt; liferay-bpm - PR#4460 - 2025-05-02[06:06:05]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>207398299</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/publish-testray-report/31040/">publish-testray-report#31040</a><br/></div></div>

igor-franca commented:
Fixed modules:
object-web
portal-security
portal-workflow-kaleo-web
workspaces

Unfixed modules:
analitycs-reports
analitycs-settings
analitycs-web 
osb-faro-web
segments-web

Error: 

```
     [exec] BUILD FAILED
     [exec] /opt/dev/projects/github/liferay-portal/build-test-analytics-cloud.xml:991: The following error occurred while executing this line:
     [exec] /opt/dev/projects/github/liferay-portal/build-test-analytics-cloud.xml:776: The following error occurred while executing this line:
     [exec] /opt/dev/projects/github/liferay-portal/build-test-analytics-cloud.xml:32: The following error occurred while executing this line:
     [exec] /opt/dev/projects/github/liferay-portal/build-test-analytics-cloud.xml:754: Sourced file: inline evaluation of: `` 				 					import com.liferay.jenkins.results.parser.JenkinsResultsParserUtil;   . . . '' : Error in method invocation: Static method pullDockerImageDependencies( java.io.File, [Ljava.lang.String;, null ) not found in class'com.liferay.jenkins.results.parser.JenkinsResultsParserUtil' : at Line: 7 : in file: inline evaluation of: `` 				 					import com.liferay.jenkins.results.parser.JenkinsResultsParserUtil;   . . . '' : JenkinsResultsParserUtil .pullDockerImageDependencies ( new File ( "/opt/dev/projects/github/com-liferay-osb-asah-private" ) , excludedImageNames .split ( "," ) , project .getProperty ( "analytics.cloud.ecr.registry" ) ) 

```

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

igor-franca commented:
resent [here](https://github.com/brianchandotcom/liferay-portal/pull/161461)
